include/lsst/afw/image/ImageBase.h

```diff
@@ -249,8 +249,9 @@ class ImageBase {
      * declare this function private
      */
     ImageBase& operator=(const ImageBase& rhs);
-    ImageBase& operator=(ImageBase&& rhs);
-    /// Set the %image's pixels to rhs
+    ImageBase& operator=(ImageBase&& rhs) noexcept;
+
+    virtual /// Set the %image's pixels to rhs
```

Comment:
This is not wanted here, and would be a weird place for it anyway.  Same for all of the ones below.

Reply:
I'll take those out. It was a mistake to do blind automatic fixing in those cases.